src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/bucket/UsageBucket.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.summaries.bucket;
+
+/**
+ * A UsageBucket is associated with a {@link com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.Resource}
+ * and identifies the state of that Resource. We use these buckets to identify when we have the
+ * bandwidth to scale a particular resource out or in.
+ *
+ * <p>{@link UsageBucket#HEALTHY_WITH_BUFFER} means that the
+ * {@link com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.Resource} is healthy and
+ * potentially under utilized. Resources in this state are good candidates for scaling in
+ *
+ * <p>{@link UsageBucket#HEALTHY} means that the
+ * {@link com.amazon.opendistro.elasticsearch.performanceanalyzer.grpc.Resource} is in a healthy
+ * state. Resources in this bucket should probably be left alone.
+ *
+ * <p>{@link UsageBucket#UNDER_UTILIZED} means that the
```

Comment:
the enum is named `UNDER_UTILIZED` in description we mention - _The utilization is high, but not quite high enough_. This can be confusing. 

It might make sense to have it in 4 levels such as `hot, warm, luke-warm, cold and unknown` or just level1, level2 etc. What do you think ?

Reply:
I hadn't properly updated my comments. I have done so, so that now things should be less confusing. I think the current bucket names are more descriptive of what we're trying to do. @vigyasharma or @rguo-aws can be a tie-breaking vote here.